Paymetheus/Send.xaml

```diff
@@ -232,7 +232,7 @@
                 <ColumnDefinition/>
             </Grid.ColumnDefinitions>
             <TextBlock Grid.Row="0" Style="{StaticResource shellMediumHeaderTextBlockStyle}" Text="Create transaction"/>
-            <TextBlock Grid.Row="1" Style="{StaticResource shellDescriptionTextBlockStyle}" Width="600" Padding="0 10" VerticalAlignment="Top">
+            <TextBlock Grid.Row="1" Style="{StaticResource shellDescriptionTextBlockStyle}" Width="575" Padding="0 10" VerticalAlignment="Top">
```

Comment:
I tried to make the text of the two windows line up as best as I could, since they're close to each other in the shell list and both are used to transfer coins. I think the sizes of the window/stackview were different, so setting them the same didn't yield similar widths.